src/lib/locals.js

```diff
@@ -39,9 +41,13 @@ module.exports = {
     const toggleValue = req.app.get("APP.LANGUAGE_TOGGLE_ENABLED");
     res.locals.showLanguageToggle = toggleValue && toggleValue === "1";
     res.locals.htmlLang = req.i18n.language;
-    res.locals.currentUrl = new URL(
-      req.protocol + "://" + req.get("host") + req.originalUrl,
-    );
+    try {
```

Comment:
I see the description says "A try-catch block has been added to CE, and additional error handling will be implemented within the package itself" - I guess this is the try catch so can I ask where and when additional error handling will be implemented?

Reply:
Within the language toggle package and a PR has been raised[ here](https://github.com/govuk-one-login/govuk-one-login-frontend/pull/112)